redash/handlers/data_sources.py

```diff
@@ -83,6 +96,12 @@ def get(self):
             except AttributeError:
                 logging.exception("Error with DataSource#to_dict (data source id: %d)", ds.id)
 
+        self.record_event({
+            'action': 'list',
+            'object_id': 'admin/data_sources',
+            'object_type': 'data_sources',
```

Comment:
The object type for this event is `datasource` below, but `data_sources` here. Should this be the same?